metadata/src/test/java/org/apache/kafka/metadata/migration/KRaftMigrationDriverTest.java

```diff
@@ -214,8 +233,9 @@ public void testOnlySendNeededRPCsToBrokers() throws Exception {
             metadataPublisher -> { },
             new MockFaultHandler("test"),
             quorumFeatures,
+            metrics,
             mockTime
-        )) {
+            )) {
```

Comment:
nit: looks like your IDE maybe reformatted some things. can you revert the whitespace and formatting changes?